packages/p2p/src/stores/floating-rate-store.js

```diff
@@ -0,0 +1,65 @@
+import { action, computed, observable } from 'mobx';
+import { ad_type } from 'Constants/floating-rate';
+import BaseStore from 'Stores/base_store';
+import { requestWS } from 'Utils/websocket';
+
+export default class FloatingRateStore extends BaseStore {
+    @observable fixed_rate_adverts_status;
+    @observable float_rate_adverts_status;
+    @observable float_rate_offset_limit;
+    @observable fixed_rate_adverts_end_date;
+    @observable exchange_rate;
+    @observable change_ad_alert;
+    @observable api_error_message = '';
+    @computed
+    get rate_type() {
+        if (this.float_rate_adverts_status === 'enabled') {
+            return ad_type.FLOAT;
+        }
+        return ad_type.FIXED;
+    }
+
+    @action.bound
+    setFixedRateAdvertStatus(fixed_rate_advert_status) {
+        this.fixed_rate_adverts_status = fixed_rate_advert_status;
+    }
+    @action.bound
+    setFloatingRateAdvertStatus(floating_rate_advert_status) {
+        this.float_rate_adverts_status = floating_rate_advert_status;
+    }
+    @action.bound
+    setFoatRateOffsetLimit(offset_limit) {
```

Comment:
```suggestion
    setFoatRateOffsetLimit(offset_limit) {
```
There seem to be a spelling error here. I think it should be `float` instead of `foat`

Reply:
will fix it. Thank you